index.md

```diff
@@ -47,6 +47,14 @@ babel-otherlangs:
 
 colorlinks: true # Colorize links, useful for the digital version
 
+pagemark-roman: true # Use roman numerals for all pagemarks
+# index-roman: true # Use roman numerals for indices in content tables
+
+footer:
+  pagemark:  # Determine where the page counter is located, default is center, options are right & left
+    left: true
+  authors: true # Include authors in footer, away from the pagemark.
```

Comment:
I don't think roman pagemarks should be enabled by default.
Also, it should probably be put into the general `footer.pagemark` section.

```suggestion
# index-roman: true # Use roman numerals for indices in content tables

footer:
  pagemark:  # Determine where the page counter is located, default is center, options are right & left
    left: true
    roman: false
  authors: true # Include authors in footer, away from the pagemark.
```